src/test/java/org/jitsi/meet/test/web/WebParticipant.java

```diff
@@ -401,7 +401,7 @@ public void waitForParticipants(int n)
         waitForCondition(
             () -> (Boolean) executeScript(
                 "return APP.conference"
-                    + ".listMembers().length >= " + n + ";"),
+                    + ".listMembers().length === " + n + ";"),
```

Comment:
I'm wondering if this could break other tests which may depend on that, but we'll see...

Reply:
It is the same test, waiting for 1 participant, no other tests use this :)

Reply:
Thanks.